Sprint-2/stretch/count-words.js

```diff
@@ -26,3 +26,29 @@
 
 3. Order the results to find out which word is the most common in the input
 */
+function countword(sentence){
+    let str = "";
+    str = sentence.replace(/[.,!?"]/g, '');
+    str = str.toLowerCase();
+    let words = str.split(" ").filter(word => word !== "");
+    let obj = {};
+    for (let item of words) {
+      obj[item] = (obj[item] || 0) + 1;
+    }
+    const keys = Object.keys(obj);
+  for (let i = 0; i < keys.length; i++) {
+    for (let j = i + 1; j < keys.length; j++) {
+      if (obj[keys[i]] < obj[keys[j]]) {
+        const temp = keys[i];
+        keys[i] = keys[j];
+        keys[j] = temp;
+      }
+    }
+  }
+  const sortedObject = {};
+  for (let key of keys) {
+    sortedObject[key] = obj[key];
+  }
+  return sortedObject;
```

Comment:
This works, but involves you having to write quite a lot of code...

Can you work out a way to do this same thing using `Object.fromEntries`, `Object.entries`, and `Array.sort`? I'm going to challenge you to do this in three lines of code.